.pre-commit-config.yaml

```diff
@@ -194,7 +194,7 @@ repos:
     additional_dependencies:
     - >-  # standard interface declarations for AWX plugins
       awx_plugins.interfaces
-      @ git+https://github.com/ansible/awx_plugins.interfaces.git@ad4a965
+      @ git+https://github.com/ansible/awx_plugins.interfaces.git@ca7802a
```

Comment:
I mentioned it somewhere already — best to drop Git references, as they are last resort.

```suggestion
    - awx_plugins.interfaces == 0.0.1a2
```

Reply:
fixed via #65